demos/point_discharge2d.py

```diff
@@ -147,9 +144,9 @@ def solver(index, ic):
 # there is no time dependence, the QoI looks just like an ``"end_time"`` type QoI. ::
 
 
-def get_qoi(mesh_seq, sol, index):
+def get_qoi(mesh_seq, index):
     def qoi():
-        c = sol["c"]
+        c = mesh_seq.fields["c"][0]
```

Comment:
I notice that in some cases index 0 is used and in others index 1. Is it the case that for steady-state problems index 0 is used and index 1 for time-dependent?

Reply:
I will get rid of this confusion in a very near-future PR, so that we do not even initialise the lagged solution for steady problems